divspl/main.py

```diff
@@ -22,7 +22,8 @@ def _bytes(x):
     if len(argv) > 1:
         with open(argv[1], 'r') as f:
             result = parser.parse(lexer.lex(f.read()))
-            os.write(1, _bytes(result.eval()))
+            for number in result.eval():
+                os.write(1, _bytes(number))
```

Comment:
If you take a look at the end of [the CI output](https://travis-ci.org/di/divspl/builds/246299353), it shows it all on one line:

```
The command "python pypy/rpython/bin/rpython --output=bin/divspl divspl/target.py" exited with 0.
0.02s$ bin/divspl fizzbuzz.divspl
12fizz4buzzfizz78fizzbuzz11fizz1314fizzbuzz
```